frontend/packages/component-library/src/section/Section.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import {HTMLAttributes, ReactNode} from 'react';
+
+import moduleStyles from './section.module.scss';
+
+export interface SectionProps extends HTMLAttributes<HTMLElement> {
+  /** Background color */
+  backgroundColor?:
```

Comment:
nit: might be worth create variable and type for colors like we did for buttons. (https://github.com/code-dot-org/code-dot-org/blob/staging/frontend/packages/component-library/src/button/Button.tsx#L6 , https://github.com/code-dot-org/code-dot-org/blob/staging/frontend/packages/component-library/src/button/types.ts#L3 )(theoretically it makes easier to support component in the future)
Definitely no need to move these type and variable to a separate file though

Reply:
Like this? https://github.com/code-dot-org/code-dot-org/pull/63706/commits/68407a690b6205b0b71d90c4d7c0ea29cc04db1b